bxserum/provider_test/integration/ws_test.go

```diff
@@ -94,11 +95,27 @@ func TestWSClient_Requests(t *testing.T) {
 				_, err := w.SubmitOrder(owner, payer, market, side, []pb.OrderType{pb.OrderType_OT_LIMIT}, amount, price, opts)
 				require.NotNil(t, err)
 
-				return err.Error()
+				return strings.Trim(err.Error(), "\"")
```

Comment:
don't think this should be necessary -- if you're seeing this it's usually because you cast the error message directly to a `string` without JSON unmarshaling. best to fix this at the source instead of covering it up here